app/controllers/alaveteli_pro/embargo_extensions_controller.rb

```diff
@@ -0,0 +1,32 @@
+# -*- encoding : utf-8 -*-
+# app/controllers/alaveteli_pro/embargo_extensions_controller.rb
+# Controller for embargo extensions
+#
+# Copyright (c) 2008 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class AlaveteliPro::EmbargoExtensionsController < AlaveteliPro::BaseController
+  def create
```

Comment:
I felt like this was more rails' "golden path" because of the simpler form builders, validation and so on. It also keeps a record of extensions to a particular embargo, which I thought might be valuable in the future, even if just for usage tracking. 

I'm not wed to it though, happy to turn it into an update instead if you'd rather..